keygen/tarsnap-keygen.1-mdoc

```diff
@@ -10,9 +10,9 @@
 .Xr tarsnap 1
 .Sh SYNOPSIS
 .Nm
-.Fl -keyfile Ar key-file
-.Fl -user Ar user-name
-.Fl -machine Ar machine-name
+.Fl -keyfile Ar filename
```

Comment:
This is a draft PR for discussion.

I've gone back and forth about "--keyfile key-file" vs. "--keyfile filename" a few times.  I like the look of the paragraph description better with "filename":
> The --keyfile **filename** option specifies the **filename** in which to write the newly-generated keys.

However, I must admit that you're pretty consistent in using `key-file` in other user-facing info (man pages, `-h` output, etc.), so it wouldn't be good to change it in just this one place.  Maybe we could compromise on using `key-filename` ?

(I must admit that a great deal of my enthusiasm for `filename` comes from when I was writing a lot of python in academia, and I was fed up with getting files and filenames confused, so I resolved to always use "filename" (or "foo_filename" when I was referring to a string, and "foo_file" for file objects.)